lib/redis_memo/memoize_method.rb

```diff
@@ -40,6 +40,24 @@ def memoize_method(method_name, method_id: nil, **options, &depends_on)
     end
 
     alias_method method_name, method_name_with_memo
+
+    @memoized_dependencies ||= Hash.new
+    @memoized_dependencies[method_name] = depends_on
+
+    define_method :memoized_dependencies do |method_name, *method_args|
+      self.class.memoized_dependencies(method_name, *method_args)
+    end
+  end
+
+  def memoized_dependencies(method_name, *method_args)
```

Comment:
I wonder what's the best name for this method. Ideally, we should make this method name clear and simple. 

How about `dependency_of`?